commands/backups/url.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function * run (context, heroku) {
+  const host = require('../../lib/host')()
+  const pgbackups = require('../../lib/pgbackups')(context, heroku)
+  const sortBy = require('lodash.sortby')
+
+  const {app, args} = context
+
+  let num
+
+  if (args.backup_id) {
+    num = yield pgbackups.transfer.num(args.backup_id)
+    if (!num) throw new Error(`Invalid Backup: ${args.backup_id}`)
+  } else {
+    let transfers = yield heroku.get(`/client/v11/apps/${app}/transfers`, {host})
+    let lastBackup = sortBy(transfers.filter(t => t.succeeded && t.to_type === 'gof3r'), 'created_at').pop()
+    if (!lastBackup) throw new Error(`No backups on ${cli.color.app(app)}. Capture one with ${cli.color.cmd('heroku pg:backups:capture')}`)
+    num = lastBackup.num
+  }
+
+  let info = yield heroku.post(`/client/v11/apps/${app}/transfers/${num}/actions/public-url`, {host})
+  cli.log(info.url)
```

Comment:
I'm fine with the change if Rimas is. "Act differently depending on whether output is tty" is not unprecedented (e.g., GNU `ls` does this for `--color=auto` which is now default--not sure about BSD `ls`), but I don't feel strongly one way or the other.